scripts/configuration.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs')
+
+const flags = [/myapp/g, /MyApp/g]
+const longNameFlags = [/My App/g]
+
+/* DIR PATHS */
+const UIPath = 'packages/ui/'
+const APIPath = 'packages/api/'
+
+const setupConfigFile = './setup.config.json'
+const AWSCredentials = (username) => `/Users/${username}/.aws/credentials`
```

Comment:
Let's add a `#TODO: add support for Windows` comment for our Windows friends. https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html